packages/auth/index.rn.ts

```diff
@@ -51,16 +51,37 @@ export { PhoneMultiFactorGenerator } from './src/platform_browser/mfa/assertions
 
 export { getReactNativePersistence };
 
+const NO_PERSISTENCE_WARNING = `
+You are initializing Firebase Auth for React Native without providing
+AsyncStorage. Auth state will default to memory persistence and will not
+persist between sessions. In order to persist auth state, install the package
+"@react-native-async-storage/async-storage" and provide it to
+initializeAuth:
+
+import { initializeAuth, getReactNativePersistence } from 'firebase/auth';
```

Comment:
Do we need to import getReactNativePersistence again? Isn't the one in line 31 sufficient?

Reply:
We're not importing it here, this is instructions to the user to import it and use it. Since we don't actually call getReactNativePersistence() ourselves anymore, I'll actually remove line 31 and make the export in line 51 export directly from the file.

Reply:
Thanks! I didnt realize this was still part of the instructions/warning.